tests/sysauth/sssd.pm

```diff
@@ -17,19 +17,34 @@ use warnings;
 use testapi;
 use utils;
 use version;
-
+use version_utils qw(is_sle is_opensuse);
+use registration "add_suseconnect_product";
 
 sub run {
     # Assume consoletest_setup is completed
     select_console 'root-console';
+    if (is_sle) {
+        assert_script_run 'source /etc/os-release';
+        if (is_sle '>=15') {
+            if (script_run('SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300) != 0) {
+                record_soft_failure 'bsc#1124318 - Fail to get PackageHub Pool Metadata - running the command again as a workaround';
+                assert_script_run 'SUSEConnect -p PackageHub/${VERSION_ID}/${CPU}', 300;
+            }
+            add_suseconnect_product('sle-module-legacy', undef, undef, undef, 300);
```

Comment:
```suggestion
            add_suseconnect_product('sle-module-legacy');
```

Reply:
@Zaoliang any particular reason why you want to set this to 5 minutes?

Reply:
@SergioAtSUSE feel free to merge with the suggestion commited or not, I'll merge later if you don't (commiting the suggestion :P )

Reply:
well, registration system takes times, it depends on network and proxy server. 5 minutes is safe :)